subsys/bluetooth/controller/ll_sw/nordic/lll/lll_central_iso.c

```diff
@@ -143,6 +171,17 @@ static int prepare_cb(struct lll_prepare_param *p)
 					   &data_chan_prn_s,
 					   &data_chan_remap_idx);
 
+	/* Store the current event latency */
+	cig_lll->latency_event = cig_lll->latency_prepare;
+	lazy = cig_lll->latency_prepare - 1U;
+
+	/* Reset accumulated latencies */
+	cig_lll->latency_prepare = 0U;
+
+	/* Adjust the SN and NESN for skipped CIG events */
+	cis_lll->sn += cis_lll->tx.bn * lazy;
+	cis_lll->nesn += cis_lll->rx.bn * lazy;
```

Comment:
So `sn` and `nesn` are 1 bits each, and `lazy` is a `uint16_t`. 

Is this designed to specifically overflow?

Reply:
Alright, please add a comment that states this. 

Could perhaps also be made more clear by doing `&= 1` as you just want the lowest bit, right?